src/lib/triggers.js

```diff
@@ -129,7 +137,10 @@ class Triggers {
 
 					route('/');
 					parentCall('openWidget');
-					store.setState({ minimized: false });
+
+					if (!window.mobilecheck()) {
```

Comment:
Instead of defining `minimized: false`, wouldn't it be better to skip calling the `parentCall` method?

Reply:
That would be better (y)

But we have to skip both parentcall and setstate({minimized: false]) to not open widget in mobiles.

Reply:
By `skipping` I meant `return`...